vaTerra/screens/chat.jsx

```diff
@@ -0,0 +1 @@
+//chat function

```

Comment:
please add a space before the word chat @Siebe88

Reply:
I am on it boss